docs/README.md

```diff
@@ -1,9 +1,21 @@
 # Documentation & Website Generation
 
-To build the documentation locally, run `make html` from the `/docs` directory and the docs site will build to: `docs/_build/html/index.html`, which can be opened in the browser.
+To build the documentation:, from the `/docs` folder:
+
+Run:
+* `python autogeneration/pages/evaluators.py` -> creates evaluator pages
+* `python autogeneration/pages/metrics.py` -> creates metrics page
+
+> Note: credoai-lens and credoai-connect needs to be installed for this to work correctly.
+
+After pushing to github, Read the Docs will build the documentation if the
+branch is currently activated.
+
+For local build, on top of the previous commands, run `make html`directory and the docs site will build to: `docs/_build/html/index.html`, which can be opened in the browser.
```

Comment:
> Should add instructions for what to do if you can't auto-build, since that's the current state of things.

@esherman-credo do you mean in case RTD build fails?

> What is the process for putting locally-built docs onto RTD manually?

You mean loading the local build directly to RTD? I don't think there is a way to do that. If the build works locally, in general it should work remotely, bar dependency issues. There shouldn't be a need for this.

> What is necessary to go back to a state where we can auto-generate/auto-push to RTD?

It would be necessary to add Lens as a dependency, which is pretty onerous for just a doc build. This is due 
to the code in the evaluators page generator being dependent on `build_list_of_evaluators` and the internal structure of the Lens package. We could find maybe some workaround, atm I think we are OK with the current manual approach.

Reply:
Ah forgive me. I was under the impression something wasn't building on RTD and that you had to upload manually. Based on discussion last week Wed or Fri.

It seems by "manual" you mean parsing the files rather than importing?

Reply:
Yeah running the parsing manually before pushing to github.